### PR TITLE
No-index the application

### PR DESCRIPTION
**Pull request by bilbof**
https://trello.com/c/Puke0ZAM/60-no-index-all-pages

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -5,6 +5,7 @@
     <title><%= yield :title %> - GOV.UK</title>
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="shortcut icon" href="https://www.gov.uk/favicon.ico" type="image/x-icon" />
+    <meta name="robots" content="noindex">
```
Comment (leenagupte) [inline app/views/layouts/application.html.erb]: Why are we noindexing?

Comment (timblair) [reply app/views/layouts/application.html.erb]: All journeys to services should start on GOV.UK: https://www.gov.uk/service-manual/technology/get-a-domain-name#telling-search-engines-not-to-index-pages

Comment (timblair) [reply app/views/layouts/application.html.erb]: But you should add `nofollow` to this directive too.

Comment (bilbof) [reply app/views/layouts/application.html.erb]: Thanks, added.

**Review by huwd — APPROVED** [review]

**Pull request merged**